templates/ep19/bs/user_panel/forms/profile_settings_picture.html

```diff
@@ -0,0 +1,58 @@
+{% load crispy_forms_tags %}
+
+<div class="col-md-8">
+    <fieldset>
+        <label class="radio row">
+            <div class="col-md-6">
+                <input type="radio" name="picture_options" value="none"{% if selected_picture_option == "none" %} checked="checked"{% endif %}> Do not show any picture
+            </div>
+        </label>
+        <label class="radio row">
+            <div class="col-md-6">
+                <input type="radio" name="picture_options" value="gravatar"{% if selected_picture_option == "gravatar" %} checked="checked"{% endif %}> Use my Gravatar
```

Comment:
Enum here would be nice as well, but caveat here is that Choices wouldn't work here, because in the template it would be resolved to verbose name value.